src/Sulu/Bundle/AdminBundle/Resources/js/components/ArrowMenu/arrowMenu.scss

```diff
@@ -15,6 +15,7 @@ $arrowMenuBoxShadow: 0 0 14px 0 rgba($doveGray, .5);
     box-shadow: $arrowMenuBoxShadow;
     overflow: auto;
     min-width: 200px;
+    max-height: 500px;
```

Comment:
Thanks for creating this pull request! Unfortunately, this solution not solve the issue for small browser windows and might waste space for big browser windows 😕